backtesting/backtesting.py

```diff
@@ -544,8 +565,9 @@ def _copy(self, **kwargs):
     def close(self, portion: float = 1., indicator: str = None):
         """Place new `Order` to close `portion` of the trade at next market price."""
         assert 0 < portion <= 1, "portion must be a fraction between 0 and 1"
-        size = copysign(max(1, round(abs(self.__size) * portion)), -self.__size)
-        order = Order(self.__broker, size, parent_trade=self, close_indicator=indicator)
+        size = copysign(max(1, abs(self.__size) * portion), -self.__size)
```

Comment:
add explanations pls with links to other places, if we need to fix it again later